eclair/src/diff.rs

```diff
@@ -43,32 +146,27 @@ fn extract_time_and_data(summary: &Summary) -> (Vec<String>, HashMap<&ItemId, us
     (times, data_item_map)
 }
 
-pub fn diff(candidate: &Summary, reference: &Summary) {
+pub fn diff(candidate: &Summary, reference: &Summary, output: Option<PathBuf>) {
```

Comment:
Look like `output` only need be a `Option<&Path>` -- client code can allocate if they need to.